resources/views/layout.blade.php

```diff
@@ -123,6 +124,11 @@ class="icon-bar"></span> <span class="icon-bar"></span></button>
                             Division Info <span class="caret"></span>
                         </a>
                         <ul class="dropdown-menu" role="menu">
+                            @if(!Auth::check())
+                                <li><a href="/tmu/notices" target="_blank">N.T.O.S. - TMU Notices</a></li>
+                                <li class="divider"></li>
+                            @else
```

Comment:
The contents of this else were left behind. (Effectively wrapping everything between here and the old location in the else unnecessarily)

We also need to handle the "TMU Maps" section (and it's sub items, those previously contained within the else). Perhaps we can move all of these links to a new page and just provide a "TMU Maps" link to it within Division Info.